fail2ban/tests/files/logs/postfix

```diff
@@ -148,6 +148,9 @@ Jan 14 16:18:16 xxx postfix/smtpd[14933]: warning: host[192.0.2.5]: SASL CRAM-MD
 # failJSON: { "time": "2005-01-14T16:18:16", "match": true , "host": "192.0.2.5", "desc": "aggressive only" }
 Jan 14 16:18:16 xxx postfix/smtpd[14933]: warning: host[192.0.2.5]: SASL CRAM-MD5 authentication failed: Invalid authentication mechanism
 
+# failJSON: { "time": "2004-11-04T09:11:01", "match": true , "host": "192.0.2.152", "desc": "reason unavailable" }
+Nov  4 09:11:01 mail postfix/smtpd[1234]: warning: unknown[192.0.2.152]: SASL LOGIN authentication failed: (reason unavailable), sasl_username=admin
```

Comment:
It'd be indeed matched, however the question is whether it is an artificially constructed message or real message example?
With other words whether the other parts remain the same in case of `reason unavailable`.

Reply:
apart from the timestamp and ip address. I pulled that log from a production system.

Reply:
... cops have got the lot naomi & leroy ...so far you 2 , your mum , Chloe I' can't wait I got sooooo much on you guys ...bring ya popcorn ... I'm taking this all the way years & years of court  , well way I see it I'll be taking all 4 inheritance..... This is exciting , how much time you've all spent on me ...   God bless I'll pray for you all ..   🖕🤛